dotCMS/src/main/java/com/dotcms/rest/api/v1/apps/Input.java

```diff
@@ -0,0 +1,42 @@
+package com.dotcms.rest.api.v1.apps;
+
+import com.dotcms.repackage.javax.validation.constraints.NotNull;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Arrays;
+
+public class Input {
+
+    @NotNull
+    private char [] value;
```

Comment:
not immutable?

Reply:
you mean final? They should probably be final

Reply:
done